beacon-chain/types/block.go

```diff
@@ -56,6 +56,22 @@ func NewGenesisBlock() (*Block, error) {
 	gb := NewBlock(nil)
 	gb.data.Timestamp = protoGenesis
 
+	genesisActiveStateHash, err := NewGenesisActiveState().Hash()
```

Comment:
Ah, I probably could've been more clear in the requirements. Notice how we [initialize the active/crystallized state](https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/blockchain/core.go#L48-L49) prior to calling this method. I was suggesting that you can simply hash the existing state objects and use that in this function.

Reply:
So you want to pass the active state and crystallized state as parameter of `NewGenesisBlock()`? Otherwise we cannot access `genesisBlock.data` since it's a private member from `types` package.

Reply:
I was suggesting passing just the hash.